content/configuration/performance.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Performance
+sort: 14
+contributors:
+  - thelarkinn
+---
+
+These options allows you to control how webpack notifies you of assets and entrypoints that exceed a specific file limit. 
+This feature was inspired by the idea of [webpack Performance Budgets](https://github.com/webpack/webpack/issues/3216).
+
+## `performance`
+
+`object`
+
+Configure how performance hints are shown. For example if you have an asset that is over 250kb, webpack will emit a warning notifiying you of this.
+
+
+## `performance.hints`
+
+`boolean | "error" | "warning"`
+
+Turns hints on/off. In addition, tells webpack to throw either an error or a warning when hints are found. This property is set to `"warning"` by default.
+
+Given an asset is created that is over 250kb:
+
+```js
+performance: {
+  hints: false
+}
+```
+
+No hint warnings or errors are shown.
+
+```js
+performance: {
+  hints: "warning"
+}
+```
+
+A warning will be displayed notifying you of a large asset. We recommend something like this for development environments.
```

Comment:
Yes true. Maybe that's where they could filter assets, etc.

I mean at the most these are recommendations but I 100% agree where you are coming from.